finish-deployment/deploy.sh

```diff
@@ -76,7 +134,11 @@ function save_files() {
             echo "IP: ${NODE_IP%%/*}"
             echo ">>>>"
             ${SSH_COMMAND} -i ${RSA_KEY_FILE} core@${NODE_IP%%/*} "mkdir -p ~/.kube"
-            copy_files_common "${SPOKE_KUBECONFIG}" "${NODE_IP%%/*}" "./.kube/config"
+            copy_files_common "${SPOKE_CSR-KUBECONFIG}" "${NODE_IP%%/*}" "./.kube/ztpfw-csr-approver-config"
+            
+            if [[ -f "~/.kube/config"} ]]; then
```

Comment:
This "if" statement is not necessary and it's wrong xD.

- Think that you are in a pod, and you need to test if the file exists in the remote server
- I think it's just better to send the command to delete the file, even if delete the remote file or not it will return 0